fvm/src/gas/price_list.rs

```diff
@@ -1093,6 +1093,16 @@ impl Rules for WasmGasPrices {
             | Operator::Return
             | Operator::Else
             | Operator::End => Ok(InstructionCost::Fixed(0)),
+            // Make casts and constants free.
+            Operator::I64ExtendI32U // only unsigned is free.
+            | Operator::I32WrapI64 // truncates
+            | Operator::I32ReinterpretF32 // casts
+            | Operator::I64ReinterpretF64 // casts
+            | Operator::V128Const { .. }
+            | Operator::I32Const { .. }
+            | Operator::I64Const { .. }
+            | Operator::F32Const { .. }
+            | Operator::F64Const { .. } => Ok(InstructionCost::Fixed(0)),
```

Comment:
@Jacarte so.... have you seen any cases where one might be able to make a very slow wasm executable with many consts? My assumption is that they'll slow down compilation, but basically get folded into instructions at runtime. But I'm not 100% sure.